model/src/main/kotlin/io/spine/validation/WithValidationSettings.kt

```diff
@@ -33,7 +33,7 @@ import io.spine.protodata.settings.defaultConsumerId
  * An interface common for components of [ValidationPlugin] that load settings
  * passed as a file containing [ValidationConfig] instance written using the plugin class name.
  */
-public interface ValidationPluginPart: LoadsSettings {
+public interface WithValidationSettings: LoadsSettings {
```

Comment:
P.4 from the PR description, namely

> 4. Renamed `ValidationPluginPart` to `WithValidationSettings`. Only implicit required policies use this type.

... hardly explains the reason.

Please elaborate.